Src/BootCamp.Chapter/ConsoleLogger.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+
+namespace BootCamp.Chapter
+{
+    public class ConsoleLogger : ILogger
+    {
+        private string _message;
+
+        public ConsoleLogger()
+        {
+        }
+
+        public ConsoleLogger(string message)
+        {
+            _message = message;
+        }
+
+        public void SetMessage(string message)
```

Comment:
Can be removed, message should be passed straight to Log method.

Reply:
Changed in [5810276](https://github.com/csinn/CSharp-From-Zero-To-Hero/pull/215/commits/58102760f5e95c0fde15237a0be1be8bf5399646)